engine/redis-helper.js

```diff
@@ -0,0 +1,24 @@
+import redis from 'redis';
+
+function getClient(dbTestNumber) {
+  const client = redis.createClient(process.env.REDISCLOUD_URL, { no_ready_check: true });
+  return new Promise((resolve, reject) => {
+    if (dbTestNumber) {
+      client.select(dbTestNumber, (err) => {
+        if (err) {
+          reject(err);
+        }
+        resolve(client);
+      });
+    } else {
+      resolve(client);
+    }
+  })
+  .catch((err) => {
+    throw new Error(err);
```

Comment:
Yes - I think it would be better.. I just c&p the function from engine/index to this helper